.vortex/tooling/src/helpers.php

```
@@ -360,6 +360,112 @@ function drush(string $command, ?int &$exit_code = NULL): string {
+  if (@file_get_contents($file, FALSE, NULL, 0, 2) !== "\x1f\x8b") {
+    return;
+  }
+
+  task('Extracting the database backup.');
+  command_must_exist('tar');
+
+  $archive = $file . '.tar.gz';
+  rename($file, $archive);
+  passthru_or_fail(sprintf('tar -xzOf %s > %s', escapeshellarg($archive), escapeshellarg($file)), 'Failed to extract the database backup.');
+  unlink($archive);
+}
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 456. (undefined)

(ErrorControlOperator)